I/ImageCore/Compat.toml

```diff
@@ -48,10 +48,15 @@ ShowItLikeYouBuildIt = "0.1-0.2"
 FFTW = "0.0-0.2"
 
 ["0.7"]
+julia = "0.7-1.3"
+
+["0.7-0.8"]
 MappedArrays = "0.2"
 OffsetArrays = "0.8-0.10"
 PaddedViews = "0.4.1-0.4.2"
-julia = "0.7-1.3"
 
-["0.7.2-0.7.4"]
+["0.7.2-0.8"]
 ColorTypes = "0.7.5"
+
+["0.8"]
+julia = "1.0-1.1"
```

Comment:
This looks wrong. cc @nkottary

Reply:
https://github.com/JuliaComputing/Registrator.jl/issues/103

Reply:
This can be re-triggered now.